invenio_users_resources/services/schemas.py

```diff
@@ -68,6 +69,21 @@ class UserProfileSchema(Schema):
     affiliations = fields.String()
 
 
+def validate_username(value):
```

Comment:
Can we use [`invenio_account.utils.validate_username`](https://github.com/inveniosoftware/invenio-accounts/blob/master/invenio_accounts/utils.py#L229)?

Reply:
Maybe we should call it and simply change the exception. The regular expression used to validate the account can be change and will lead to inconsistencies https://github.com/inveniosoftware/invenio-accounts/blob/master/invenio_accounts/config.py#L321.